habitat_baselines/common/obs_transformers.py

```diff
@@ -754,15 +754,15 @@ def __init__(self, equ_h: int, equ_w: int):
         input_projections = get_cubemap_projections()
 
         # Equirectangular output
-        output_projection = EquirecCamera(equ_h, equ_w)
-        super(Cube2Equirec, self).__init__(
+        output_projection = EquirectProjection(equ_h, equ_w)
+        super(Cube2Equirect, self).__init__(
             input_projections, output_projection
         )
 
 
-# TODO Measure Inheritance of CubeMap2Equirec + CubeMap2FishEye into same abstract class
+# TODO Measure Inheritance of CubeMap2Equirect + CubeMap2FishEye into same abstract class
```

Comment:
Maybe we should make a ProjectionTransformer here and have all these things subclass it.